frontend/model/chatroom/getters.js

```diff
@@ -128,7 +135,14 @@ const getters: { [x: string]: (state: Object, getters: { [x: string]: any }, roo
     }
   },
   isJoinedChatRoom (state, getters, rootState) {
-    return (chatRoomID: string, memberID?: string) => !!rootState[chatRoomID]?.members?.[memberID || getters.ourIdentityContractId]
+    return (chatRoomID: string, memberID?: string) => {
+      return getters.isJoinedChatRoomForChatRoom(rootState[chatRoomID], memberID)
+    }
+  },
+  chatRoomActiveMemberIds (state, getters, rootState) {
+    return (chatRoomID: string) => {
+      return getters.chatRoomActiveMemberIdsForChatRoom(rootState[chatRoomID])
+    }
```

Comment:
This new getter has been added but is never used anywhere. So are you sure it should be here?

Reply:
Would you rather I remove or rename this getter?

Reply:
If it followed the convention, it'd use `currentChatRoomState`, which would actually mean we could use it in `ChatMixin`.